spec/discourse_code_review/lib/github_pr_syncer_spec.rb

```diff
@@ -89,6 +89,9 @@ def get_user_email(github_login)
     )
   end
 
+  before { User.set_callback(:create, :after, :ensure_in_trust_level_group) }
+  after { User.skip_callback(:create, :after, :ensure_in_trust_level_group) }
```

Comment:
A little bit ugly, to avoid this ugly: https://github.com/discourse/discourse/blob/main/spec/rails_helper.rb#L244-L245

This is because the plugin creates new users deep within its belly, and they need the auto-groups for the tag creation not to fail.